ament_copyright/ament_copyright/__init__.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I pulled this from Apache's tempate: https://www.apache.org/licenses/LICENSE-2.0 

I think we should use their template, not remove the 4 whitespace.